docs/developers/contributing.md

```diff
@@ -29,12 +29,14 @@ Bugfixes should include a regression test which prevents that bug from being re-
 
 ## Update Documentation
 
-Documentation for the project is stored in the [docs](..) directory. If your feature requires docs updates,
-those updates must be submitted in the same Pull Request as the code changes.
+Documentation for the project is stored in the prebid-server directory on the [Prebid repo](https://github.com/prebid/prebid.github.io/tree/master/prebid-server). If your feature requires docs updates,
+those updates must be submitted in the same Pull Request as the code changes. Documentation can be viewed at 
```

Comment:
Shouldn't this: `If your feature requires docs updates,
those updates must be submitted in the same Pull Request as the code changes` sentence be updated to say that the doc updates need to be submitted in a _separate_ PR on the [prebid.github.io](https://github.com/prebid/prebid.github.io) repo?